src/averageAges.js

```diff
@@ -20,6 +20,19 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+
+  const manCheck = person => person.sex === 'm';
+  const centuryCheck = person => {
+    return manCheck(person) && Math.ceil(person.died / 100) === century;
+  };
+
+  const men = century !== undefined
+    ? people.filter(centuryCheck)
+    : people.filter(manCheck);
+
+  const menAvrAge = getAverageAge(men);
+
+  return menAvrAge;
```

Comment:
you don't have to create separate variable
```suggestion
  return getAverageAge(men);
```